### PR TITLE
Update ComponentPatcher

### PR DESCRIPTION
**Pull request by Strencher**
PR to fix things (listed below) in the `renderer/src/modules/componentpatcher.js`.

The following things got updated/fixed/added:
- `Utilities.onAdded` util.
- Guild Utility classes
- BD dev chat badge
- BD profile badge

### CODE REVIEW

File: renderer/src/modules/componentpatcher.js [hunk 1]
```diff
@@ -4,27 +4,47 @@ import DiscordModules from "./discordmodules";
 import Utilities from "./utilities";
 import Patcher from "./patcher";
 import BDLogo from "../ui/icons/bdlogo";
+import Logger from "../../../common/logger";
```
Comment (Strencher) [inline renderer/src/modules/componentpatcher.js:7]: Well, i guess it should be `common/logger` instead?

Comment (rauenzi) [reply renderer/src/modules/componentpatcher.js:7]: It should be, yes

File: renderer/src/modules/componentpatcher.js [hunk 2]
```diff
@@ -109,42 +164,62 @@ export default new class ComponentPatcher {
         });
     }*/
 
+    /**
+     * @updated 07.07.2021
```
Comment (rauenzi) [inline renderer/src/modules/componentpatcher.js:168]: Are these needed?

File: renderer/src/modules/componentpatcher.js [hunk 3]
```diff
@@ -75,10 +123,17 @@ export default new class ComponentPatcher {
             if (guildData.video) returnValue.props.className += " bd-video";
             if (guildData.badge) returnValue.props.className += " bd-badge";
             if (guildData.animatable) returnValue.props.className += " bd-animatable";
+            if (guildData.unavailable) returnValue.props.className += " bd-unavailable";
+            if (guildData.screenshare) returnValue.props.className += " bd-screenshare";
+            if (guildData.liveStage) returnValue.props.className += " bd-liveStage";
```
Comment (rauenzi) [inline renderer/src/modules/componentpatcher.js:128]: Can you make the class `bd-live-stage` or `bd-livestage` to keep consistent with the other classes?

Comment (Strencher) [reply renderer/src/modules/componentpatcher.js:128]: Sure

File: renderer/src/modules/componentpatcher.js (hunk 2 above)
Comment (Strencher) [reply renderer/src/modules/componentpatcher.js:168]: They are good for reference to know when it was updated. I can remove them if it bothers you.

**Pull request merged**